<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>K210BurnDialog</class>
 <widget class="QDialog" name="K210BurnDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>470</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>K210Burn</string>
  </property>
  <widget class="QGroupBox" name="groupBoxFileInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>491</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>ChooseFile</string>
   </property>
   <widget class="QLineEdit" name="lineEditFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Target FileName</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFileAddress">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>Target Address</string>
    </property>
    <property name="text">
     <string notr="true">0x000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpen">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Browser Target Files</string>
    </property>
    <property name="text">
     <string>Browser...</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonDownload">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Click to start Download</string>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxUartInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>491</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Uart Configure</string>
   </property>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Device</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelBaud">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Baudrate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="CustomComboBox" name="comboBoxPort">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>335</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="toolTip">
     <string>Target Device</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxBaud">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>335</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>Write Flash Buadrate</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>491</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar::chunk {
    background-color: (0,81,137);
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="progressLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>491</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Click to Start Download...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxEraseSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>491</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>EraseSettings</string>
   </property>
   <widget class="QLabel" name="labelEraseMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxEraseMode">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>335</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Erase Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEraseAddr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Addr</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEraseAddr">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>335</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Erase Start Address</string>
    </property>
    <property name="text">
     <string notr="true">0x000000</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEraseSize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEraseSize">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Erase Size</string>
    </property>
    <property name="text">
     <string notr="true">4</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxEraseSize">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Erase Size Unit</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEraseTemplate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Template</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxEraseTemplate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Erase Template</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadTemplate">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Load Erase Template</string>
    </property>
    <property name="text">
     <string>LoadTemplate</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonSwitchMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Switch Work Mode</string>
   </property>
   <property name="text">
    <string>SwitchMode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonErase">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Click to Start Erase</string>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomComboBox</class>
   <extends>QComboBox</extends>
   <header>openmv/k210_burn/custom_combobox/custom_combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
